client/components/blazepress-widget/style.scss

```diff
@@ -77,6 +80,54 @@ $headerHeight: 72px;
 			margin-top: 0;
 		}
 	}
+
+	&.blazepress-i2 {
+		.blazepress-widget__header-bar {
+			justify-content: flex-start;
+			width: calc(100% - 40px);
+			min-height: 32px;
+			height: auto;
+			padding: 24px 20px;
+			margin: 0;
+
+			.wordpress-logo {
+				width: 2rem;
+				height: 2rem;
+			}
+
+			.blank-canvas__back {
+				padding: 0;
+				height: 2rem;
+				font-family: $heading-font-family;
```

Comment:
According to Figma, the font size should be `15px` as for the "Powered by Jetpack" sign.